plugins/jira-dashboard/src/components/JiraDashboardContent/JiraDashboardContent.tsx

```diff
@@ -85,7 +88,7 @@ export const JiraDashboardContent = (props?: { annotationPrefix?: string }) => {
             </Grid>
             {jiraResponse.data.map((value: JiraDataResponse) => (
               <Grid data-testid="issue-table" key={value.name} md={6} xs={12}>
-                <JiraTable tableContent={value} />
+                <JiraTable tableContent={value} showFilters />
```

Comment:
Currently, this makes showFilters always true in JiraTable. Should be showFilters={props?.showFilters}.